force/force.go

```diff
@@ -51,6 +51,7 @@ func Create(version, clientId, clientSecret, userName, password, securityToken,
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
?

Reply:
Code cleanup. Nicer to look at with the newline.